docs/source/command_line.rst

```diff
@@ -554,6 +554,36 @@ of the above sections.
     Note: the exact list of flags enabled by running :option:`--strict` may change
     over time.
 
+.. option:: --disable-error-code
+
+    This flag allows disabling one or multiple error codes globally.
+
+    .. code-block:: python
+
+        # no flag
+        x = 'a string'
+        x.trim()  # error: "str" has no attribute "trim"  [attr-defined]
+
+        # --disable-error-code attr-defined
+        x = 'a string'
+        x.trim()
+
+.. option:: --enable-error-code
+
+    This flag allows enabling one or multiple error codes globally.
+
+    NOTE: Will override disabled error codes from the --disable-error-code
```

Comment:
Nist: For consistency, don't use all caps (above we use "Note: ..."). Use a full sentence, such as "This will ...".